torch/fx/experimental/normalize.py

```diff
@@ -80,13 +83,13 @@ def call_module(self, target : Target, args : Tuple[Argument, ...], kwargs : Dic
             classname = submod.__class__.__name__
             if getattr(torch.nn, classname, None) == submod.__class__:
                 sig = inspect.signature(inspect.unwrap(submod.forward))
-                new_kwargs = self._args_kwargs_to_normalized_kwargs(sig, args, kwargs)
-                if new_kwargs:
-                    return super().call_module(target, (), new_kwargs)
+                new_args, new_kwargs = self._args_kwargs_to_normalized_args_kwargs(sig, args, kwargs)
+                if new_args is not None and new_kwargs is not None:
+                    return super().call_module(target, new_args, new_kwargs)
         return super().call_module(target, args, kwargs)
 
-    def _args_kwargs_to_normalized_kwargs(self, sig : inspect.Signature, args : Tuple[Argument, ...],
-                                          kwargs : Dict[str, Any]) -> Optional[Dict[str, Any]]:
+    def _args_kwargs_to_normalized_args_kwargs(self, sig : inspect.Signature, args : Tuple[Argument, ...],
+                                          kwargs : Dict[str, Any]) -> Optional[Tuple[Tuple[Any], Dict[str, Any]]]:
```

Comment:
This return type is getting pretty complex. Can we define a class or NamedTuple for the return?